sdk/tables/data-tables/src/TableBatch.ts

```diff
@@ -246,7 +246,6 @@ function parseBatchResponse(batchResponse: HttpOperationResponse): TableBatchRes
 /**
  * Prepares the operation url to be added to the body, removing the SAS token if present
  * @param url - Source URL string
- * @returns
```

Comment:
There are many more occurrences for this. Look for `@returns$` after enabling regex and excluding the `arm-*` folders